src/test/java/core/basesyntax/ArrayListTest.java

```diff
@@ -27,7 +27,7 @@
 
     @Test
     public void addElementToArrayListByIndex() {
-        ArrayList<String> arrayList = new ArrayList<>();
+        ArrayList<String> arrayList = new ArrayList<>(size);
```

Comment:
The same issue as above; the variable 'size' is being used without being declared or initialized.